tests/performance-test/grafana/datasource.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: 1
+
+datasources:
+- name: SAFElasticsearch
+  type: elasticsearch
+  access: proxy
+  orgId: 1
+  url: "https://elasticsearch.sa-telemetry.svc:9200"
+  database: "performance-test"
+  basicAuth: false
+  withCredentials: true
+  isDefault: false
+  jsonData:
+    esVersion: 5
+    keepCookies: []
+    maxConcurrentShardRequests: 256
+    timeField: "startsAt"
+    tlsAuth: true
+    tlsAuthWithCACert: true
+  secureJsonData:
+    tlsCACert:     ${CACERT}
+    tlsClientCert: ${CLIENTCERT}
+    tlsClientKey:  ${CLIENTKEY}
+  version: 1
+  editable: true
+
+- name: SAFPrometheus
+  type: prometheus
+  access: proxy
+  orgId: 1
+  url: "http://prometheus-operated.sa-telemetry.svc:9090"
+  database: ""
+  basicAuth: false
+  withCredentials: false
+  isDefault: false
+  jsonData:
+    timeInterval: "1s"
```

Comment:
This sets the lower limit on the "step" parameter for queries, which protects you from misreading your silly things when your queries demand a higher resolution than prometheus actually has. Having too low a value also causes higher CPU demand during prometheus queries as it fills in data from further back in the db. See this issue[1] for a good set of hints on this value and the potential problems.

Ideally it should be set to whatever the prometheus scrape interval is set to (see: https://github.com/redhat-service-assurance/smart-gateway-operator/pull/19) The easiest fix is probably to query this from Prometheus and re-use the value in a datasource template, but it's somewhat tricky because it can actually have different scrape intervals for different targets; choosing the global default might be best? 

This isn't too critical for our purposes so I wouldn't hold the PR up for it unless we observe it doing bad things. When grafana becomes part of SAF proper I think this will actually be easier to solve.

[1] https://github.com/prometheus/prometheus/issues/2564

Reply:
Whoops, this comment belonged on the prometheus datasource. Not sure what to set it to for the ES datasource. 1s? ;)